integration-tests/deployment/ccip/add_lane_test.go

```diff
@@ -1 +1,57 @@
 package ccipdeployment
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+)
+
+// TestAddLane covers the workflow of adding a lane
+// between existing supported chains in CCIP.
+func TestAddLane(t *testing.T) {
+	// TODO: The offchain code doesn't yet support partial lane
+	// enablement, need to address then re-enable this test.
```

Comment:
What is needed still for offchain?

Reply:
nothing now if the IsEnabled flag work is done, then we can complete this test